### PR TITLE
[EOS-19396]-Ensure dependent services are started for S3Server

### PR DESCRIPTION
**Pull request by tanujashinde0405**
Signed-off-by: Tanuja Shinde <tanuja.shinde@seagate.com>

### CODE REVIEW

File: srv/components/s3server/config/prepare.sls [hunk 1]
```diff
@@ -15,25 +15,19 @@
...
+  service.running:
+    - name: sshd
+    - enable: True
+
 Stage - Prepare S3server:
   cmd.run:
     - name: __slot__:salt:setup_conf.conf_cmd('/opt/seagate/cortx/s3/conf/setup.yaml', 's3:prepare')
     - failhard: True
 
-# {% import_yaml 'components/defaults.yaml' as defaults %}
```
Comment (83bhp) [inline srv/components/s3server/config/prepare.sls:23]: Why are we removing the sections to add s3 repos?

Comment (tanujashinde0405) [reply srv/components/s3server/config/prepare.sls:23]: It was a part of EOS-18921 - Explode config files.
https://github.com/Seagate/cortx-prvsnr/pull/1070

**Review by 83bhp — APPROVED** [review]
One minor question on the changes to understand the patch, rest looks ok.

File: srv/components/s3server/config/prepare.sls [hunk 2]
```diff
@@ -15,25 +15,19 @@
...
 
+include:
+  - components.misc_pkgs.rsyslog.start
+  - components.ha.haproxy.start
+  - components.misc_pkgs.openldap.start
+
+Start sshd service:
+  service.running:
+    - name: sshd
+    - enable: True
```
Comment (ypise) [inline srv/components/s3server/config/prepare.sls:26]: Don't enable here. Leave it in it's set state, just ensure it's running.

**Review by ypise — APPROVED** [review]

**Pull request merged**